tests/storage_test.go

```diff
@@ -142,6 +142,45 @@ var _ = Describe("Storage", func() {
 
 			runHostPathJobAndExpectCompletion(pod)
 		}
+		Context("with NFS that will be deleted", func() {
+			It("should pause VMI on IO error", func() {
+				nfsPod := initNFS(tests.HostPathAlpine)
+				pvName := createNFSPvAndPvc(k8sv1.IPv4Protocol, nfsPod)
+
+				vmi := tests.NewRandomVMIWithEphemeralPVC(pvName)
+
+				vmi = tests.RunVMIAndExpectLaunchWithIgnoreWarningArg(vmi, 180, true)
+				Expect(console.LoginToAlpine(vmi)).To(Succeed())
+
+				By("Deleting NFS")
+				err := virtClient.CoreV1().Pods(tests.NamespaceTestDefault).Delete(context.TODO(), nfsPod.Name, metav1.DeleteOptions{})
+				Expect(err).NotTo(HaveOccurred())
+				Eventually(ThisPod(nfsPod), 30*time.Second, time.Second).Should(BeGone())
+
```

Comment:
Is the PV automatically deleted when the `nfsPod` is gone? The VMI pod still uses the PVC, so k8s should protects the PVC till is in use?

Reply:
Are we talking about the PV created from nfs? This part should kill the nfs and simulate the ioerror. The k8s don't protect the nfs. Nonetheless, I want to base this test on this [PR](https://github.com/kubevirt/kubevirt/pull/5457)

Reply:
Oh, I see. I have also tried nfs in order to trigger the io error. However, libvirt is not aware that the storage is on a nfs server. That's the reason I think it doesn't trigger the io error :)

Reply:
I had to read from the file on the disk to trigger it(this can be flaky).